Parsers/ASimAuditEvent/Tests/VMwareCarbonBlackCloud_AsimAuditEvent_SchemaTest.csv

```diff
@@ -0,0 +1,90 @@
+﻿Result
+"(1) Warning: Missing recommended field [Dst]"
+"(1) Warning: Missing recommended field [DvcDomain]"
+"(1) Warning: Missing recommended field [DvcHostname]"
+"(1) Warning: Missing recommended field [DvcIpAddr]"
+"(1) Warning: Missing recommended field [EventResultDetails]"
+"(1) Warning: Missing recommended field [ObjectId]"
+"(1) Warning: Missing recommended field [TargetHostname]"
+"(1) Warning: Missing recommended field [TargetIpAddr]"
+"(2) Info: Missing optional alias [Application] aliasing non-existent column [TargetAppName]"
+"(2) Info: Missing optional alias [Process] aliasing non-existent column [ActingProcessName]"
+"(2) Info: Missing optional field [ActingAppId]"
+"(2) Info: Missing optional field [ActingAppName]"
+"(2) Info: Missing optional field [ActingAppType]"
+"(2) Info: Missing optional field [ActorOriginalUserType]"
+"(2) Info: Missing optional field [ActorScopeId]"
+"(2) Info: Missing optional field [ActorScope]"
+"(2) Info: Missing optional field [ActorSessionId]"
+"(2) Info: Missing optional field [ActorUserAadId]"
+"(2) Info: Missing optional field [ActorUserId]"
+"(2) Info: Missing optional field [ActorUserSid]"
+"(2) Info: Missing optional field [DvcDescription]"
+"(2) Info: Missing optional field [DvcFQDN]"
+"(2) Info: Missing optional field [DvcId]"
+"(2) Info: Missing optional field [DvcInterface]"
+"(2) Info: Missing optional field [DvcMacAddr]"
+"(2) Info: Missing optional field [DvcOriginalAction]"
+"(2) Info: Missing optional field [DvcOsVersion]"
+"(2) Info: Missing optional field [DvcOs]"
+"(2) Info: Missing optional field [DvcScopeId]"
+"(2) Info: Missing optional field [DvcScope]"
+"(2) Info: Missing optional field [DvcZone]"
+"(2) Info: Missing optional field [EventCount]"
```

Comment:
Pls update this field in tester for Audit and File. Its Class should be Mandatory instead of Optional
https://github.com/Azure/Azure-Sentinel/blob/master/ASIM/dev/ASimTester/ASimTester.csv

Reply:
Updated